google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -416,28 +424,30 @@ def create_table table_id, name: nil, description: nil, fields: nil
         #   bigquery = Google::Cloud::Bigquery.new
         #   dataset = bigquery.dataset "my_dataset"
         #   view = dataset.create_view "my_view",
-        #             "SELECT name, age FROM [proj:dataset.users]"
+        #             "SELECT name, age FROM proj.dataset.users"
         #
         # @example A name and description can be provided:
         #   require "google/cloud/bigquery"
         #
         #   bigquery = Google::Cloud::Bigquery.new
         #   dataset = bigquery.dataset "my_dataset"
         #   view = dataset.create_view "my_view",
-        #             "SELECT name, age FROM [proj:dataset.users]",
+        #             "SELECT name, age FROM proj.dataset.users",
         #             name: "My View", description: "This is my view"
         #
         # @!group Table
         #
-        def create_view table_id, query, name: nil, description: nil
+        def create_view table_id, query, name: nil, description: nil,
```

Comment:
We can make this more error proof by making sure that standard_sql and legacy_sql are mutually exclusive parameters.

Calling this function with both fields set is incorrect and we should protect against that.